effekt/jvm/src/main/scala/effekt/Server.scala

```diff
@@ -3,14 +3,13 @@ package effekt
 import effekt.context.Context
 import effekt.core.PrettyPrinter
 import effekt.lifted.LiftInference
-import effekt.source.{ FunDef, Hole, ModuleDecl, Tree }
-import effekt.util.{ PlainMessaging, getOrElseAborting }
+import effekt.source.{FunDef, Hole, ModuleDecl, Tree}
+import effekt.util.{PlainMessaging, getOrElseAborting}
```

Comment:
Please try to keep this changes out of the PR or change them upstream, so that the diff is smaller :)